torch/_inductor/lowering.py

```diff
@@ -920,7 +920,10 @@ def squeeze(x, dim=None):
 
     new_shape = []
     for d, s in enumerate(x.get_size()):
-        if not (d in dims and V.graph.sizevars.evaluate_expr(sympy.Eq(s, 1))):
+        if not (
+            d in dims
+            and V.graph.sizevars.evaluate_expr(sympy.Eq(s, 1, size_oblivious=True))
```

Comment:
did you need `size_oblivious=True` as part of this PR?

Reply:
Yes, it is needed for symbolic shapes processing of the testcase for NJT